test/integration/resultsPageErrors.js

```diff
@@ -0,0 +1,80 @@
+const cheerio = require('cheerio');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+const iExpect = require('../lib/expectations');
+const messages = require('../../app/lib/messages');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+const resultsRoute = `${constants.SITE_ROOT}/results/`;
+
+function assertSearchResponse(search, assertions) {
```

Comment:
I realise there is duplication here with the resultsPage tests but it is just an interim step before we use beforeEach.